frontend/src/metabase/parameters/components/ParameterSidebar/ParameterSidebar.tsx

```diff
@@ -123,6 +125,13 @@ export const ParameterSidebar = ({
     [otherParameters],
   );
 
+  const handleChangeRequired = useCallback(
```

Comment:
afaik ParametersSidebar will re-render when you select another parameter, so parameterId will change. 

because of that I would drop useCallback as unnecessary until you found unnecessary re-renderings without it

Reply:
Thanks, removed. I actually looked at the other callbacks and just did the same for the sake of consistency. But you're right, it's unnecessary.